App.js

```diff
@@ -25,9 +25,26 @@ const firebaseConfig = {
   appId: "1:158582890553:web:2ef50439fcd80f2ede2de8",
 };
 
-firebase.initializeApp(firebaseConfig);
+if (!firebase.apps.length) {
```

Comment:
If you are on firebase 9 please checkout master for a more up to date reference:
https://github.com/mobinni/MealsToGo/blob/master/App.js